src/tests/tests/first_parties.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  QUnit.test("test twitter content script", (assert) => {
+    const NUM_CHECKS = 1;
+    let done = assert.async(NUM_CHECKS);
+
+    // create element as we'd get it from twitter
+    let tco = "http://t.co/beach-detour/";
+    let destination = "https://the.beach/";
+    let el = document.createElement("div");
+    el.setAttribute("href", tco);
+    el.setAttribute("data-expanded-url", destination);
+    document.querySelectorAll = function () {
+      return [el];
+    };
+
+    // load the content script
+    let script = document.createElement("script");
+    script.src = "../js/first_parties/twitter.js";
+    script.onload = function () {
+      unwrapTwitterURLs();
```

Comment:
Also, our linter is catching `unwrapTwitterURLs` here as being undefined, which short-circuits and fails all automated tests.

In this case we have a false positive that you can tell ESLint to ignore via the globals directive at the top of the file (`/* globals unwrapTwitterURLs:false */`), but this line should get removed so it won't be necessary.

You should get set up for local linting so that you can catch these errors/warnings early. Your best bet should be to set up your editor to lint as you save (or even as you type).